Update managing-environments.md (#1053)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/environments.md

In practice, this means that most of the details in a target will be consistent across all targets, except for the `schema` and user credentials. If you have multiple dbt users writing code, it often makes sense for _each user_ to have their own _development_ environment. A pattern we've found useful is to set your dev target schema to be `dbt_<username>`. User credentials should also differ across targets so that each dbt user is using their own data warehouse user.

> ## Deleting environments

Reviewer: ```suggestion
## Delete environments
```

> Deleting an environment will not automatically delete the associated job(s). Before you completely delete your environment, first delete any jobs associated to that environment. Once you've deleted the job(s), you can completely delete the environment. 

Reviewer: ```suggestion
Deleting an environment doesn't automatically delete its associated job(s). To completely delete your environment, first delete all jobs associated with that environment. Then, delete the environment. 
```

Reviewer: suggs: 

- for conciseness
- present tense (vs future tense)

In practice, this means that most of the details in a target will be consistent across all targets, except for the `schema` and user credentials. If you have multiple dbt users writing code, it often makes sense for _each user_ to have their own _development_ environment. A pattern we've found useful is to set your dev target schema to be `dbt_<username>`. User credentials should also differ across targets so that each dbt user is using their own data warehouse user.

> 

Reviewer 2: OOh I like this!
